Person/Person/main.m

```diff
@@ -1,63 +1,122 @@
 //
 //  main.m
-//  Person
+//  inClassPersonAssignment
 //
-//  Created by Michael Kavouras on 6/21/15.
-//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//  Created by Jovanny Espinal on 6/23/15.
+//  Copyright (c) 2015 Jovanny Espinal. All rights reserved.
 //
 
 #import <Foundation/Foundation.h>
 
-@interface Person: NSObject
+@interface Person : NSObject
 
-- (void)setName:(NSString *)name;
-- (NSString *)name;
+-(void)setCity:(NSString *)city;
+-(NSString *)city;
 
-- (void)setCity:(NSString *)city;
-- (NSString *)city;
+-(void)setPhoneNUmber:(NSString *)phoneNumber;
+-(NSString *)phoneNumber;
+
+-(void)setName:(NSString *)name;
+-(NSString *)name;
+
+-(void)changePersonsName:(Person *)aPerson
+                  toName:(NSString *)newName;
+
+-(BOOL)checkSameCity:(Person *)aPerson;
+
+-(Person *)haveChild;
 
-- (void)setPhoneNumber:(NSString *)phoneNumber;
-- (NSString *)phoneNumber;
 
 @end
 
 @implementation Person {
-    NSString *_name;
-    NSString *_phoneNumber;
     NSString *_city;
+    NSString *_phoneNumber;
+    NSString *_name;
 }
 
-- (void)setName:(NSString *)name {
-    _name = name;
+-(void)setCity:(NSString *)city{
+    _city = city;
+}
+-(NSString *)city{
+    return _city;
 }
 
-- (NSString *)name {
-    return _name;
+-(void)setPhoneNUmber:(NSString *)phoneNumber{
+    _phoneNumber = phoneNumber;
+}
+-(NSString *)phoneNumber{
+    return _phoneNumber;
 }
 
-- (void)setCity:(NSString *)city {
-    _city = city;
+-(void)setName:(NSString *)name{
+    _name = name;
+}
+-(NSString *)name{
+    return _name;
 }
 
-- (NSString *)city {
-    return _city;
+-(void)changePersonsName:(Person *)aPerson
+                  toName:(NSString *)newName{
+    [aPerson setName:newName];
 }
 
-- (void)setPhoneNumber:(NSString *)phoneNumber {
-    _phoneNumber = phoneNumber;
+-(BOOL)checkSameCity:(Person *)aPerson{
```

Comment:
This is 100% correct. Suggestion to make it more concise: Just return the result of the `isEqualToString` method call...

```objective-c
- (BOOL)checkSameCity:(Person *)aPerson {
   return [[aPerson city] isEqualToString: [self city]];
}
```